### PR TITLE
Removing Feebacks

### LINKED ISSUE #496: Remove Feedbacks
Feedbacks is an old feature that is no longer used by anyone. The support forum has replaced it, and added much more functionality.

### PR DESCRIPTION
**Pull request by crocodoyle**
Feedbacks are no longer used. Fixes #496

### CODE REVIEW

**Comment by crocodoyle** [comment]
I have discovered that the last Feedback that was left was five years ago, and regular users have the ability to delete anyone else's feedback. I don't think this feature is useful any more

**Comment by prioux** [comment]
I've changed my mind and I think removing feedbacks is a good idea.

Let's fix this PR (it has one conflict it seems) and move forward with it.

**Comment by prioux** [comment]
Missing new schema.rb

**Comment by prioux** [comment]
There are still references to Feedback model inside:

* `models/user.rb`
* `views/portal/portal_log.html.erb`

Also, adjust layout so that footers don't put 'POWERED BY CBRAIN' and 'CREDITS' on two lines.

**Comment by prioux** [comment]
@crocodoyle please address my comments of March 10.

File: BrainPortal/db/schema.rb [hunk 1]
```diff
@@ -291,15 +283,17 @@
     t.string   "time_zone"
     t.string   "service"
     t.string   "login"
-    t.string   "comment"
```
Comment (prioux) [inline BrainPortal/db/schema.rb]: This scema is wrong, it includes modification from another pull request

Comment (crocodoyle) [inline BrainPortal/db/schema.rb]: I don't know why my schema.rb is always wrong :S

File: BrainPortal/db/schema.rb [hunk 2]
```diff
@@ -291,15 +283,17 @@
...
     t.string   "confirm_token"
     t.boolean  "confirmed"
     t.string   "approved_by"
     t.datetime "approved_at"
-    t.datetime "created_at",    :null => false
-    t.datetime "updated_at",    :null => false
-    t.string   "admin_comment"
+    t.datetime "created_at",                       :null => false
+    t.datetime "updated_at",                       :null => false
+    t.text     "admin_comment"
```
Comment (prioux) [inline BrainPortal/db/schema.rb]: You still have unrelated spurious changes in the schema here.

Comment (prioux) [inline BrainPortal/db/schema.rb]: * type of `admin_command`
* added `user_id`
* added `hidden`
